express/blocks/faq/faq.js

```diff
@@ -35,15 +67,23 @@ function decorateFAQBlocks($block) {
     const $accordion = createTag('div', { class: 'faq-accordion' });
     $block.append($accordion);
 
-    const $questionDiv = createTag('div', { class: 'faq-question' });
+    const $questionDiv = createTag('h3', { class: 'faq-question' });
     $accordion.append($questionDiv);
     $questionDiv.innerHTML = question;
 
     const $answerDiv = createTag('div', { class: 'faq-answer' });
     $accordion.append($answerDiv);
     $answerDiv.innerHTML = answer;
+
+    appendToSchema($schemaScript, question, answer);
   });
 
+  if (showSchema !== 'no') {
+    document.head.appendChild($schemaScript);
+  } else {
+    $schemaScript.remove();
+  }
+
```

Comment:
Do authors really need to be able to turn the schema off?

Reply:
That's what they said in the Jira issue. I wonder the same. We can maybe bring that up to PMs later today?

Reply:
Yes. I think it's adding unnecesary complexity if not really needed.